spec/ApplicationPattern+services.yaml

```diff
@@ -55,11 +55,6 @@ servers:
       own-ip-address: ## TODO: Add OwnIpAddress
```

Comment:
tcp-server is represented as a list(a hyphen in the starting). Similar to http-server , this shall be represented as an object(without hyphen)

Reply:
Well actually, the modelling is still a list, despite we agreed to put just one instance of the object.
If you would agree, I would prefer leaving that one as is until we changed the modelling back towards a single object.

Reply:
Sure , let's leave it as it is. Thanks.